### PR TITLE
Makes the brass skewer more ouchie than ouchie~

### PR DESCRIPTION
**Pull request by ghost**
## Changelog

🆑 
tweak: Makes the brass skewer force the user to say something instead of moan
/:cl:

i literally just did this in browser, if i need to fix it i'll do it later at home

### CODE REVIEW

**Comment by Sishen1542** [comment]
+1

**Comment by Ghommie** [comment]
Absolutely based, my friend.

**Comment by Useroth** [comment]
At least add a couple of possible speech lines lines.

**Comment by ghost** [comment]
Nice

**Comment by ghost** [comment]
HOW DID IT FAIL THE TRAVIS CHECK

File: code/modules/antagonists/clockcult/clock_structures/traps/brass_skewer.dm [hunk 1]
```diff
@@ -106,7 +106,7 @@
 		"<span class='danger'>You start tenderly lifting [skewee] off of [src]...</span>")
 		if(!do_after(user, 60, target = skewee))
 			skewee.visible_message("<span class='warning'>[skewee] painfully slides back down [src].</span>")
-			skewee.emote("moan")
+			skewee.forcesay("Oof, ouch owwie!!")
```
Comment (Ghommie) [inline code/modules/antagonists/clockcult/clock_structures/traps/brass_skewer.dm]: `skewee` is a living mob arg of this proc, `forcesay()` is a human proc, mostly used to force people to blurt out what they were typing followed by an appe-GLORF!

```suggestion
			skewee.say("Oof, ouch owwie!!", forced = "fail brass skewer removal")
```

Or cut-paste forcesay() proc down to living/say.dm and make it a living proc ez pz.

**Comment by ghost** [comment]
time to wait one month for merge lol

**Comment by BlueWildrose** [comment]
The skewer made me shout this in dead chat after I got myself killed from a skewer, amusingly enough. Probably a minor bug of sorts.

**Review by silicons — APPROVED** [review]

**Pull request merged**